sycl/plugins/unified_runtime/ur/usm_allocator.cpp

```diff
@@ -885,11 +622,11 @@ size_t Bucket::Capacity() {
   if (getSize() <= ChunkCutOff())
     return 1;
   else
-    return USMSettings.Capacity[getMemType()];
+    return OwnAllocCtx.getParams().Capacity;
```

Comment:
how we will handle different values for Capacity?

Reply:
Please see my previous answer